app/loader.h

```diff
@@ -75,19 +109,24 @@ class Loader: public QObject
     void loadingStarted();
     void loadingFinished();
 
+    void mapSettingsChanged();
+
   public slots:
     void appStateChanged( Qt::ApplicationState state );
     // Reloads project if current project path matches given path (its the same project)
     bool reloadProject( QString projectDir );
+
   private:
     QList<QgsExpressionContextScope *> globalProjectLayerScopes( QgsMapLayer *layer );
     QgsProject *mProject = nullptr;
     QgsQuickPositionKit *mPositionKit = nullptr;
     bool mRecording = false;
 
     MapThemesModel &mMapThemeModel;
-    LayersModel &mLayersModel;
     AppSettings &mAppSettings;
+    ActiveLayer &mActiveLayer;
+    LayersProxyModel &mLayersProxyModel;
```

Comment:
Now that we don't use the proxy model, let's remove it from Loader...

Reply:
Sure!

Reply:
done